.github/actions/e2e-atomic/action.yml

```diff
@@ -1,5 +1,14 @@
 name: "Atomic E2E tests"
 description: "Runs Cypress tests for Atomic"
+inputs:
+  cypress-record-key:
+    description: The Cypress record key for actions that must be recorded
+    required: false
+    default: ''
```

Comment:
These actions are also used in `prbot.yml`, which doesn't record and therefor doesn't need secrets.